src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/host-form/host-form.component.html

```diff
@@ -32,6 +32,38 @@
           </div>
         </div>
 
+        <!-- Address -->
+        <div class="form-group row">
+          <label class="cd-col-form-label"
+                 for="addr"
+                 i18n>Nework address</label>
+          <div class="cd-col-form-input">
+            <input class="form-control"
+                   type="text"
+                   placeholder="172.10.2.144"
```

Comment:
https://stackoverflow.com/questions/10456044/what-is-a-good-invalid-ip-address-to-use-for-unit-tests

Reply:
You meant to add unit tests or change the placeholder? Right now i changed the placeholder